.stylelintrc

```diff
@@ -28,5 +28,6 @@
     ],
     "csstree/validator": true
   },
-  "ignoreFiles": ["build/**", "dist/**", "**/reset*.css", "**/bootstrap*.css"]
-}
+  "ignoreFiles": ["build/**", "dist/**", "**/reset*.css", "**/bootstrap*.css"],
+  "files": ["**/*.css", "**/*.scss"]
```

Comment:
- [ ] Great job reverting all the commented linter codes. However, you're not allowed to modify any of the linter files. Kindly find a way to resolve whatever linter issue you have. You might only restricted yourself to the allowed tools to have your way.